thoth/storages/graph/postgres.py

```diff
@@ -4116,6 +4118,7 @@ def _create_python_package_version(
         python_version: Union[str, None],
         python_package_metadata_id: int = None,
         sync_only_entity: bool = False,
+        import_package: str,
```

Comment:
mandatory parameter cannot be after optional one :) is it optional? then I would put `Optional[str] = None`

Reply:
thank you)
I think it should be mandatory in this context 
corrected with c7996586173df63d59a6d46ac5a5063bc89bca20